Controller/TemplatingExceptionController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Templating\EngineInterface;
+use Symfony\Bundle\FrameworkBundle\Templating\TemplateReference;
+use Symfony\Component\HttpFoundation\Request;
+
+abstract class TemplatingExceptionController extends ExceptionController
+{
+    protected $templating;
+
+    public function __construct(
+        ExceptionWrapperHandlerInterface $exceptionWrapperHandler,
+        FormatNegotiator $formatNegotiator,
+        ViewHandlerInterface $viewHandler,
+        array $exceptionCodes,
+        array $exceptionMessages,
+        $showException,
+        EngineInterface $templating
+    ) {
+        parent::__construct($exceptionWrapperHandler, $formatNegotiator, $viewHandler, $exceptionCodes, $exceptionMessages, $showException);
```

Comment:
Well, I see one drawback with that solution: We needed to do runtime checks then to ensure that we can actually access the value which adds a bit overhead for (in my opinion) no added value. And if someone really wants to use the class outside the bundle, they will need to go the hard way to figure out which methods to call to properly set up the object.